src/Lib/Models/CanvasRect.vala

```diff
@@ -89,11 +89,19 @@ public class Akira.Lib.Models.CanvasRect : Goo.CanvasRect, Models.CanvasItem {
     }
 
     public void update_border () {
+        //We only use one value (don't do ellipses)
         if (is_radius_uniform) {
             set ("radius-x", radius_x);
             set ("radius-y", radius_x);
+        } else {
```

Comment:
Use a `return;` here instead of an `else`.

Reply:
Yes, let me release that on a goocanvas-3.0 branch so we get any other improvement for free.